app/assets/stylesheets/drilldown.scss

```diff
@@ -1,3 +1,30 @@
 // Place all the styles related to the drilldown controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+#check-button {
```

Comment:
This is fine for now, but it is always good to be aware of when we are duplicating code.

If you prefer, we can either put this css in a new file and `import` it, or simply move it into the `applications.scss` file which will be read by all templates.